source/redirect-latest.html.slim

```diff
@@ -0,0 +1,5 @@
+- content_for(:senarios) do
```

Comment:
You must be in a different time zone.  It's 11 PM here and I'll be going to bed shortly :).